### ISSUE TITLE
Exported frozen graph of PSPNet50

### ISSUE THREAD

**Issue by CalciferZh** [oandrienko/fast-semantic-segmentation#15]
Hi, thank you for your work. I'm trying to load your pre-trained weights to another graph as initialization. I found that in the frozen graph there's not `gamma` or `beta` variable under BatchNorm, but only one constant named `Conv2D_bn_offset`. Is this because the convolution and the batch normalization are merged in the frozen graph? Thank you in advance.
